osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -90,17 +90,14 @@ public partial class ScoreProcessor : JudgementProcessor
         private readonly double accuracyPortion;
         private readonly double comboPortion;
 
-        /// <summary>
-        /// Scoring values for a perfect play.
-        /// </summary>
-        public ScoringValues MaximumScoringValues
+        public Dictionary<HitResult, int> MaximumStatistics
         {
             get
             {
                 if (!beatmapApplied)
                     throw new InvalidOperationException($"Cannot access maximum scoring values before calling {nameof(ApplyBeatmap)}.");
 
-                return maximumScoringValues;
+                return maximumResultCounts.ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
```

Comment:
I'm guessing that a full copy of the dictionary was intended here (to get a fresh reference). If that was the case, you can also write

```suggestion
                return new Dictionary<HitResult, int>(maximumResultCounts);
```